### ISSUE TITLE
Review Design Wireframe: News Post page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#96]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by EliNygard** [comment]
- The content is not aligned to the grid. Is quite far into the page, on the second column in the grid
![Skjermbilde 2024-04-04 142506](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/142724042/8ef97a20-3145-4779-b8c5-b9bd4cdc04dd)

- layout

**Comment by EliNygard** [comment]
Review finished and comments added for the Design Team

**Comment by Vickabu** [comment]
*Since footer and nav is mention i several issues, but with different comments - it should have been one issue with one precise description. Same with the newsletter. and the arrows *
*Alternativ font is used here as mentioned in Style Guide - but we will update Style Guide to be more clear*

**Comment by SiljeW** [comment]
_ "Desktop and mobile view: the line height on the paragraphs is too narrow. Should be 1.4 to 1.6 times larger than the font size"_
- The paragraphs are adjusted and font sizes are now compatible with styleguide

_"Mobile view: the icons 'like' and 'share', and the button 'Read next blog post' are aligned with the grid, but the text paragraph, titles and images above are further in on the grid and not aligned."_
- Everything is adjusted to fit the grid

_"I miss something at the top of the blog post that tells me that this is a blog post. Like a date or author. A thought. Should be discussed with client or content team."_
- It was described earlier on, but we have edited it to make author and publishing date more clear

**Comment by EliNygard** [comment]
Great, then I'll close this.
